manifest.json

```diff
@@ -12,19 +12,19 @@
     },
```

Comment:
Signature for `...caUq`

`CGfV/PzY/anL8aLJF9UuRXmO6aqAFGZ5b1qcE2vdoo1xv5qnctcV6J3CyPMGdGuG8Y3FaSPPjwH2X7NMghM6DQ==`